.github/workflows/test-report.yaml

```
@@ -0,0 +1,46 @@
+name: test-report
+
+on:
+  schedule:
```

It might be worth temporarily running on this PR to make sure everything is configured correctly 

```suggestion
  # TODO: remove `pull_request:` condition below before merging
  pull_request:
  schedule:
```

---

I suspect it still won't work as I don't think pull requests get `GITHUB_TOKEN`s with the ability to push to the main repo (in this case a `gh-pages` branch.

But I could upload it as an artifact so we can manually download and open it (a previous commit on the branch had that, and I removed it for this PR). It just wouldn't test the git-ops-iest part of pushing to `gh-pages`

---

Ah, make sense. That would work too. Mostly it'd just be good to see the report gets generated successfully

---

How are github pages currently configured for the org? I see that `dask.github.io` redirects to the `dask.org`.

---

Demonstration CI run, with artifact upload instead of the deploy step: https://github.com/ian-r-rose/distributed/actions/runs/1560280158

---

Good point. Currently we have https://github.com/dask/dask.github.io configured to redirect to dask.org. It's been a while since I've used GitHub pages, does that mean the test report here will be available at dask.github.io/distributed  (which will redirect to dask.org/distributed)?

---

To be honest, I'm not sure. On my account it's available at `ian-r-rose.github.io/distributed`, even though I have a separate repo for `ian-r-rose.github.io`. So there is some dynamic dispatching at some level

---

Relevant docs: https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#types-of-github-pages-sites